e2e/user-story-2.spec.ts

```diff
@@ -1,7 +1,7 @@
 // File ignored in ESLint
 // Reason: the `no-await-in-loop` error broke Detox's synchronization
 import { by, device, expect, element, waitFor } from "detox";
-import { BuildingId } from "../src/types/main";
+import { getCampusById } from "../src/constants/campus.data";
```

Comment:
The way this is imported is inconsistent with `campus-toggle.component` and doesn't seem to be using the index file

Reply:
Had to do it since detox throws an unexpected error that is arguably unrelated to the test
![image](https://user-images.githubusercontent.com/33991121/78845983-30fceb80-79d8-11ea-9576-90c229885689.png)

Reply:
It would be like so `import { getCampusById } from "../src/constants";`

and changing that 1 import statement causes an error?

Reply:
Because floors.data.ts executes a `require` statement, which jest seems to not like. I think its better to keep that import as is.

Reply:
Alright resolved.